### DISCUSSION TITLE
[WP3844 Discussion]:

### DISCUSSION THREAD

**Discussion by KestUrba** [wikipathways/wikipathways-help#51] [General]
### Discussion

There is the lack
- of splitting mTor gene product  into complexes mTor1 and mTor2
- mTor1, mTor2 connection to polyamines by the green pointer

ref. [https://medlineplus.gov/genetics/gene/mtor/](url)
ref. **Ray RM, Bavaria M, Johnson LR. Interaction of polyamines and mTOR signaling in the synthesis of antizyme (AZ). Cell Signal. 2015 Sep;27(9):1850-9. doi: 10.1016/j.cellsig.2015.06.002. Epub 2015 Jun 17. PMID: 26093026; PMCID: PMC4502979.**

### Should this pathway be deleted?

- [ ] Yes, this pathway should be deleted

**Comment by KestUrba** [comment]
one more ref. Mortazavi M, Moosavi F, Martini M, Giovannetti E, Firuzi O. Prospects of targeting **PI3K/AKT/mTOR pathway in pancreatic cancer.** Crit Rev Oncol Hematol. 2022 Aug;176:103749. doi: 10.1016/j.critrevonc.2022.103749. Epub 2022 Jun 18. PMID: 35728737.

**Comment by khanspers** [comment]
WP3844 has been updated:
- Added details for mTORC1 / mTORC2 complexes
- Updated title to better reflect that the figure/paper that the model was originally based describes prostate cancer.

As mentioned in https://github.com/wikipathways/wikipathways-help/discussions/47, I think the connection between mTor and polyamines is better suited for a separate, new pathway.
